egs/wsj/s5/steps/nnet3/chain2/train.sh

```diff
@@ -190,7 +190,7 @@ while [ $x -lt $num_iters ]; do
             --print-interval=10  \
            "nnet3-copy --learning-rate=$lrate $dir/${x}.raw - |" $den_fst_dir \
            "ark:nnet3-chain-copy-egs $egs_opts scp:$egs_dir/${name}_subset.scp ark:- | nnet3-chain-merge-egs $multilingual_eg_opts --minibatch-size=1:64 ark:- ark:-|" \
-           $dir/${next_x}_${name}.mdl || touch $dir/.error_diagnostic &
+           $dir/${next_x}_${name}.mdl || touch $dir/.error_diagnostic
```

Comment:
Not 100% ideal from speed perspective - perhaps make it an option?  [could do it as '&' and then just wait if the option is a certain way..] Open to making it like this by default.  Dont know how fast it is...

Reply:
Yes, now should be better